### ISSUE TITLE
Find way to fix despawning

### ISSUE THREAD

**Issue by Kamsta99** [kyranf/robotarmyfactorio#135]
Pls look to discusion section on factorio mods
https://mods.factorio.com/mod/robotarmy/discussion

**Comment by kyranf** [comment]
Hey! Don't worry i'm not ignoring you (well, sort of, for now). i've been working 12-14 hours a day for the last few weeks straight and I can't spare any time to deal with all the great suggestions and potential fixes you have found. 

I will get to them hopefully next week! I will have some time off then.

**Comment by Kamsta99** [comment]
Nah, I just think you miss my suggestions and fix because mod portal have no notifications ;). I want to tell you, i find way after massive playtest to balance droid to make not OP, and more additional. Also maked custom graphic for droid assembler. 
![droid-assembler](https://user-images.githubusercontent.com/8656801/37460475-9d8ef47c-284b-11e8-94c1-d80ba577a4cf.png)
 Feel free to use my graphic if you like. Also i make pipes invisible, but arrows still exit.
![20180315123254_1](https://user-images.githubusercontent.com/8656801/37460971-2c09c8f2-284d-11e8-8149-95a33ac93d4e.jpg)
Tell me i send you my code for pipes.
Also i get inspiration to change droid assembler from assembling machine to furnace to dynamically select droid recipe. Also this change fix problem with dummy droids recipes show in player crafting menu.

Its just me, i always like change some small thing whitch i didnt like ;D

**Comment by kyranf** [comment]
haah cool graphics. I have new graphics I am adding made by YuokiTani so the next update should have some nice new graphics. 

Please send me the code for pipes, I was having problems making it work the other day.

**Comment by kyranf** [comment]
>  inspiration to change droid assembler from assembling machine to furnace to dynamically select droid recipe. Also this change fix problem with dummy droids recipes show in player crafting menu.

Yes I have been wanting to do this for a long time!

**Comment by Kamsta99** [comment]
Code for pipes
[my_code.zip](https://github.com/kyranf/robotarmyfactorio/files/1852240/my_code.zip)

**Comment by kyranf** [comment]
did you end up changing the droid assembler to an electric furnace to just spawn whatever is on the input?  I will be doing this soon if you haven't done it already.

**Comment by Kamsta99** [comment]
Sorry for long reply (somehow i miss this email), but no i dont finish my
project because of low amount of time

śr., 15 sie 2018 o 16:38 kyranf <notifications@github.com> napisał(a):
